contracts/src/token/erc721/extensions/burnable.rs

```diff
@@ -3,7 +3,7 @@
 use alloy_primitives::{Address, U256};
 use stylus_sdk::msg;
 
-use crate::token::erc721::{Erc721, Error};
+use crate::token::erc721::{self, Erc721, Error};
```

Comment:
```suggestion
use crate::token::erc721::{self, Erc721};
```
Same as in previous comment

Reply:
Resolved